### PR TITLE
Bug 1112484 - [Settings] Show/hide FDN settings based on the

### PR DESCRIPTION
**Pull request by EragonJ**
availability

### CODE REVIEW

**Comment by try-server-hook** [comment]
[EragonJ EragonJ (E.J.) (EragonJ)](https://github.com/EragonJ) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=56887043b061)

File: apps/settings/js/call.js [hunk 1]
```diff
@@ -143,6 +144,18 @@ require([
       cs_refreshCallSettingItems();
     }
 
+    function cs_initFdnItem() {
+      var iccObj = getIccByIndex();
+      var fdnMenuItem = document.getElementById('menuItem-callFdn');
+      iccObj.getServiceState('fdn').then(function(hasFdn) {
+        if (hasFdn) {
```
Comment (crh0716) [inline apps/settings/js/call.js]: nit: `fdnMenuItem.hidden = !hasFdn;`.

Comment (EragonJ) [inline apps/settings/js/call.js]: ok thanks Arthur !!

**Comment by try-server-hook** [comment]
[EragonJ EragonJ (E.J.) (EragonJ)](https://github.com/EragonJ) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=b362fa93c74f)

**Pull request merged**